### PR TITLE
Recently migrated my Fulcrum (BCHN) server from DevOps to APECS.

### PR DESCRIPTION
**Pull request by nyusternie**
My old Fulcrum server ran on `fulcrum.devops.cash`, but went offline due to a full SSD. 
Everything has been upgraded to the latest 1.9.0 and moved to a new home on Vultr.

### CODE REVIEW

File: resources/bch/servers.json [hunk 1]
```diff
@@ -144,5 +144,11 @@
         "s": "50002",
         "t": "50001",
         "version": "1.4.4"
+    },
+    "fulcrum.apecs.dev": {
+        "pruning": "-",
+        "s": "50002",
+        "t": "50001",
+        "version": "1.9.0"
```
Comment (cculianu) [inline resources/bch/servers.json:152]: The version is protocol version not server version. Should be `1.5.0` or something.

**Comment by nyusternie** [comment]
oh ok.

how can I confirm the "protocol" version that I'm currently supporting? i'm upgraded to the latest server version `1.9.0`, so can I just assume that it's the latest protocol as well `1.5.0`?

`./Fulcrum -v` outputs:

```
Fulcrum 1.9.0 (Release feb8f6a)
compiled: gcc 8.4.0
jemalloc: version 5.2.1-0-gea6b3e9
Qt: version 5.15.6
rocksdb: version 6.14.6-ed43161
simdjson: version 0.6.0
ssl: OpenSSL 1.1.1  11 Sep 2018
zmq: libzmq version: 4.3.3, cppzmq version: 4.7.1
```

**Comment by cculianu** [comment]
Oh yeah hmm. I should add that as output to the `--version` command. Yes, you support both 1.4 through 1.5 on 1.9.0.

**Review by nyusternie — COMMENTED** [review]
fixed!

not to be different from the rest of the listings, but I thought i'd identify support for the latest version `1.5.0` of the protocol .. however, whatever works for you, works for me, just happy to support the network :wink:

**Comment by cculianu** [comment]
Yeah that's fine.  Note I just pushed a commit to make `--version` tell you actually what protocol version your Fulcrum "speaks".

**Comment by cculianu** [comment]
Looks good to me!

**Comment by nyusternie** [comment]
sweet! :+1:

**Pull request merged**